ledger/src/blockstore_db.rs

```diff
@@ -1451,33 +1451,45 @@ where
         result
     }
 
-    pub(crate) fn multi_get_bytes<I>(&self, keys: I) -> Vec<Result<Option<Vec<u8>>>>
+    /// Create a key type suitable for use with multi_get_bytes() and
```

Comment:
I mention "key type" here because I plan on changing this from `Vec<Vec<u8>>` so some custom type that is a single `Vec<u8>` that provides an iterator for key byte slices. That is followup for another PR